internal/config/config.go

```diff
@@ -16,7 +16,7 @@ type Config struct {
 var config *Config
 
 const (
-	_ = string(iota) // We don't care about the values of these constants
+	_ = string(rune(iota)) // We don't care about the values of these constants
```

Comment:
Just curious what does this do?

Reply:
It's a way to generate constants that have a specific, unique, but unimportant value. What we're using it for is to be able to refer to config options using a specific `const` like `Port` rather than just a string like `"port"` which allows us to be more precise about our type checking, avoid typos, etc. 
I had to add `rune` in there because something changed in go 1.15 which caused a warning due to the conversion from int directly to string. It doesn't really matter in our situation because, again, the value itself is unimportant, as long as it's a unique string.